content/c++/encyclopedia/exceptions-in-cpp.md

```diff
@@ -0,0 +1,133 @@
+---
+Title: "Exceptions in C++"
+Subjects:
+  - "Computer Science"
+  - "Game Development"
+Tags:
+  - "Catch"
+  - "Errors"
+  - "Try"
+Catalog Content:
+  - "https://www.codecademy.com/learn/learn-c-plus-plus"
+  - "https://www.codecademy.com/learn/paths/computer-science"
+  - "https://github.com/Codecademy-Curriculum/codepedia-content/blob/main/content/language-agnostic/encyclopedia/what-is-an-error.md"
+---
+
```

Comment:
Missing an introduction here - a basic one or two sentences of "what is an en exception in C++".